modules/govuk/manifests/apps/email_alert_service/rabbitmq.pp

```diff
@@ -82,8 +117,8 @@
   govuk_rabbitmq::consumer { $amqp_user:
     ensure               => $ensure,
     amqp_pass            => $amqp_pass,
-    read_permission      => "^(amq\\.gen.*|${amqp_major_change_queue}|${amqp_unpublishing_queue}|${amqp_exchange})\$",
-    write_permission     => "^(amq\\.gen.*|${amqp_major_change_queue}|${amqp_unpublishing_queue})\$",
-    configure_permission => "^(amq\\.gen.*|${amqp_major_change_queue}|${amqp_unpublishing_queue})\$",
+    read_permission      => "^(amq\\.gen.*|${amqp_major_change_queue}|${amqp_unpublishing_queue}|${ampq_subscriber_list_update_minor_queue}|${ampq_subscriber_list_update_major_queue}${amqp_exchange})\$",
```

Comment:
we didn't talk about these in our catchup, but looking at the others I think this is needed too?

Reply:
Yes, it gives the email-alert-service rabbitmq user access to those queues